go.mod

```diff
@@ -190,7 +191,18 @@ replace (
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20240510131258-f646d5f29250
 )
 
-// This replacement can be removed once we bump to k8s.io@v1.30.x, as from release 1.30+
-// https://github.com/kubernetes/code-generator/blob/release-1.30/go.mod, code-generator will use
-// the new v2 version of kube-openapi.
-replace k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20231010175941-2dd684a91f00
+replace (
```

Comment:
maybe add a TODO that we remove this for 1.16/1.36?

Reply:
Yep, that's a great idea. Done!